Makefile

```diff
@@ -4,19 +4,21 @@
 # You can set these variables from the command line.
 PYTHON        = python3
 VENVDIR       = ./venv
-BUILDDIR      = _build
-SPHINXOPTS    = -W --keep-going
 SPHINXBUILD   = $(VENVDIR)/bin/sphinx-build
+SPHINXOPTS    = -W --keep-going
+BUILDDIR      = _build
 BUILDER       = html
-SPHINXLINT    = $(VENVDIR)/bin/sphinx-lint
+JOBS          = auto
```

Comment:
`JOBS          = auto` and the `-j $(JOBS)` below are the only actual additions.  While I was at it I tried to reorganize the vars a bit to make them closer to `cpython`'s `Doc/Makefile`, including the order of the vars below.